netmiko/eltex/eltex_esr_ssh.py

```diff
@@ -16,19 +18,27 @@ def session_preparation(self):
         time.sleep(0.3 * self.global_delay_factor)
         self.clear_buffer()
 
-    def config_mode(self, config_command="configure", pattern=r")#"):
+    def config_mode(
+        self,
+        config_command: str = "configure",
+        pattern: str = r")#",
+        *args: Any,
+        **kwargs: Any
+    ) -> str:
```

Comment:
Instead of passing *args/**kwargs, I would pass this one additional argument: `re_flags: int = 0` and then use that in the super call i.e. add `re_flags=re_flags` as part of the super call.